### ISSUE TITLE
[Errata] OAK-FFC-CBA - DM0260 R0M0E0

### ISSUE THREAD

**Issue by Erol444** [luxonis/depthai-hardware#292]
We have found that the OAK-FFC-CBA (DM0260) revision R0M0E0 contains a hardware issue which results in camera not being able to run. **Please send an email to support@luxonis.com for the OAK-FFC camera board replacement**.

## Hardware solution

There's an easy hardware fix for this - you need to cut/lift pin 4 of the `U3` voltage regulator: 
![image](https://user-images.githubusercontent.com/18037362/185666272-b997b993-a2e3-4069-abdd-da76e47d1415.png)

Here you can see how we cut the pin 4, which makes the camera work as expected:
![image](https://user-images.githubusercontent.com/18037362/185666601-5da07858-be2a-47a3-8ad9-6b5e5dc0491c.png)

![image](https://user-images.githubusercontent.com/18037362/185666547-a84e7740-a1fc-46eb-a8d8-9fec2c2a6cef.png)

**Comment by Luxonis-David** [comment]
TL;DR on error explanation:

DM0260_R0M0E0 Printed Circuit Boards were produced with an error on gerbers, resulting in 2V8 regulator not working. The issue manifested only in the latest batch by using alternative part for LDK120C28R. 

The issue is in fourth pin on U3 part on PCBA being connected to GND instead left floating. We didn't detect it on first control batch using TLV70028DCK part, as the operation of the later mentioned regulator is not interrupted by this error. 
Updated design files R1M0E0 were added to GitHub which could help to ease location the issue PR is [here](https://github.com/luxonis/depthai-hardware/pull/293). 
All future boards will be manufactured with based on the R1M0E0 design files removing the issue.

**Comment by Luxonis-David** [comment]
Merged to master, project folder location [DM0260_OAK-FFC-CBA](https://github.com/luxonis/depthai-hardware/tree/master/DM0260_OAK-FFC-CBA).
